canonical/prometheus-k8s-operator
# Add scheme to alertmanagers config section

## Issue
The `alertmanagers` section in prometheus's config is rendered without a scheme and without `tls_config`, and prometheus uses `http` by default.


## Solution
- [x] Add scheme to alertmanager config.
- [ ] ~~Figure out how to populate the `tls_config` section with a `ca_file` that matches alertmanager.~~

This PR goes in tandem with https://github.com/canonical/alertmanager-k8s-operator/pull/171.

Reference: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tls_config

## Context
https://github.com/canonical/cos-lite-bundle/issues/75


## Testing Instructions
- Pack alertmanager and prom from the two PRs
- Deploy and relate prom to alertmanager, and alertmanager to self-signed-certificates
- Confirm alerts are firing on prom's `/api/v1/alerts` and that the same alerts show in alertmanagers `/api/v2/alerts`.


## Release Notes
Add scheme to alertmanagers config section.
